.travis.yml

```diff
@@ -3,13 +3,15 @@ language: php
 php:
   - 5.6
   - 7.0
+  - 7.1
+  - 7.2
 
 install:
   - composer install --prefer-source
 
 script:
-  - php vendor/bin/parallel-lint ./src
```

Comment:
The script is prefixed with `php` because it is then easier for Windows CMD users to simply copy and run the command.
In Windows CMD running `vendor/bin/parallel-lint ./src` would not work. You need Unix shell that understands   [Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) for this. This could lead to unnecessary confusion.

Too bad it is not included in `README.md` too. It was probably written by somebody else who did not realize this. :/

Reply:
Oh, I didn't know that. I thought this is now standard on Windows as well.

Do you think that  Windows users would be confused by that? I imagine they're used to it and know what you've just written. So they prefix commands by default.

I suppose they already had to use comopser, so seing `composer` would mean `php composer` for them. Or am I wrong?

Reply:
In `travis.yml` it doesn't make sense to add these, since it *nix environment only.

I could revert that in `README.md` if you think Windows users are not aware of it.

What do you think?

Reply:
Actually, Composer installer for Windows adds `composer.bat` to your `PATH` file that makes the command work without the `php` prefix.

The `php` prefix you added in "README: add Windows copy-paste support" commit solves this for me. 👍 

(Even though the commit message could be a bit misleading. Something like "README: make commands platform independent" would be better.)

Reply:
(I probably tried to include the motivation behind the commit to the message. I agree "platform independ" sounds more proffesional, will use that next time).

This is resolve then?

Reply:
Yes it is.